modules/sdk/src/main/scala/org/tessellation/sdk/cli/CliMethod.scala

```diff
@@ -52,6 +53,8 @@ trait CliMethod {
     )
   )
 
+  val collateralConfig = CollateralConfig(Amount(25000000000000L))
```

Comment:
Please change it accordingly to @tbekas suggestion using `_` or even to keep it in more human-readable format like: `250_000L * NormalizationFactor`

Reply:
Changed.